app/src/sass/_Landing.scss

```diff
@@ -0,0 +1,209 @@
+@import url('https://fonts.googleapis.com/css?family=Raleway:300i,500,700,800');
+
+#landing {
+  font-family: 'Raleway', sans-serif;
+  font-size: 18px;
+  font-weight: 500;
+  line-height: 30px;
+  color: #2e4C66;
+  -webkit-font-smoothing: antialiased;
+}
+
+#landing h1,
+#landing h2,
+#landing h3,
+#landing h4 {
+  text-transform: capitalize;
+  color: #2e4C66;
+}
+
+#landing h1 {
+  font-size: 57px;
+  font-weight: 800;
+  line-height: 68px;
+}
+
+#landing h2 {
+  font-size: 35px;
+  font-weight: 800;
+  line-height: 50px;
+}
+
+#landing h3 {
+  font-size: 27px;
+  font-weight: 500;
+  line-height: 38px;
+}
+
+#landing h4 {
+  font-size: 18px;
+  font-weight: 700;
+}
+
+#landing a {
+  color: #2FAFDE;
+  text-decoration: none;
+}
+
+#landing a:hover {
+  text-decoration: underline;
+}
+
+#landing .container {
+  margin: 0 auto;
+  text-align: center;
+  max-width: 500px;
+}
+
+#landing .btn {
+  padding: 1em 2em;
+  background-color: #2FAFDE;
+  color: #ffffff;
+  border: none;
+  border-radius: 100px;
+  box-shadow: 0px 2px 2px 1px rgba(0, 0, 0, 0.1);
+  transition: all .2s ease;
+  cursor: pointer;
+}
+
+#landing .btn:hover {
+  background-color: #2E4C66;
+}
+
+#landing a.btn:hover {
+  text-decoration: none;
+}
+
+#landing .btn.btn-secondary {
+  color: #2FAFDE;
+  background-color: #ffffff;
+}
+
+#landing .btn.btn-secondary:hover {
+  color: #2FAFDE;
+  background-color: #D5F6FF;
+}
+
+.btn-cta {
+  margin: 1em 0;
+  display: inline-block;
+}
+
+#landing .section {
+  padding: 2em;
+  position: relative;
+  text-align: center;
+}
+
+#landing .section-secondary {
+  color: #ffffff;
+}
+
+#landing .section-secondary::before {
+  content: '';
+  position: absolute;
+  left: 0;
+  top: -5px;
+  width: 100%;
+  height: 100%;
+  z-index: -1;
+  background: #2FAFDE;
+  transform: skewY(1.5deg);
+  -webkit-backface-visibility: hidden;
+  backface-visibility: hidden;
+}
+
+#landing .section-tertiary {
+  background-color: #D5F6FF;
+  color: #ffffff;
+}
+
+#landing .section-tertiary::before {
+  content: '';
+  position: absolute;
+  left: 0;
+  top: -5px;
+  width: 100%;
+  height: 50%;
+  z-index: -1;
+  background: #2FAFDE;
+  transform: skewY(-1.5deg);
+  -webkit-backface-visibility: hidden;
+  backface-visibility: hidden;
+}
+
+#landing .header-light {
+  color: #fff;
+}
+
+#landing em {
+  font-size: 18px;
+  line-height: 30px;
+  font-weight: 300;
+  font-style: italic;
+}
+
+#landing .icon-voice {
+  max-width: 50%;
+  margin: 2em auto;
+}
+
+.screens-container {
```

Comment:
This part is especially **gross**. I wanted to overlap the screen preview images over the confetti background image. 

I'm really bad at css so I couldn't figure out how to make overlaying images responsive. I will try to fix this later.